client/components/nav-bar.jsx

```diff
@@ -1,17 +1,51 @@
 import React from 'react';
 
 export default class Navbar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      drawerOpened: 'drawer-close'
+    };
+    this.handleDrawer = this.handleDrawer.bind(this);
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleDrawer(event) {
+    let drawerStatus = '';
+    this.state.drawerOpened === 'drawer-open'
+      ? drawerStatus = 'drawer-close'
+      : drawerStatus = 'drawer-open';
```

Comment:
Instead of doing assignments in the consequence expressions, do the assignment as the result of the ternary

```js
    const drawerStatus = this.state.drawerOpened === 'drawer-open'
      ? 'drawer-close'
      : 'drawer-open';
```